### PR TITLE
Write the about page content.

### PR DESCRIPTION
**Pull request by Beanow**
image

### CODE REVIEW

**Comment by Beanow** [comment]
cc @teamopen-dev/wg-organization

File: content/en/about/_index.md [hunk 1]
```diff
@@ -2,10 +2,48 @@
...
 description: |
-    Team Open --about stuff here--
+    Team Open directly supports maintainers of Open Source Software.
 ---
 
-# Why?
+## Current status: ideas
+
+We're just getting started with ideas. Team Open was
+called into being September 13th 2019 and we're now
```
Comment (nothingismagick) [inline content/en/about/_index.md:11]: Was actually International Programmers Day (and Friday the 13th fwiw)

File: content/en/about/_index.md [hunk 2]
```diff
@@ -2,10 +2,48 @@
...
+For the long term, we _spread the message_ and hope to
+make clear, this is a serious problem we need to
+address at a societal level.
+
+### What about a discussion platform?
+
+These exist, so while Team Open will likely have a good
+amount of people to have in depth discussions with,
+the primary goal is around moving into action.
+
```
Comment (nothingismagick) [inline content/en/about/_index.md:46]: Change of voice from ‘we’ to ‘I’

**Review by nothingismagick — COMMENTED** [review]
I can give you a PNGCRUSHED version of the meme image, if you want.

**Comment by nothingismagick** [comment]
I also don’t know about the centered text...

File: content/en/about/_index.md (hunk 2 above)
Comment (nothingismagick) [reply content/en/about/_index.md:46]: Dunno why this got shifted, on mobile. It refers to the next line.

**Comment by Beanow** [comment]
> I also don’t know about the centered text...

How I love css attributes are reimplemented with classes. `.tc-l { text-align: center; }`
Means I either need to override a "center text" class to be left align... or replace the template.

Either way, I'm ok leaving it centred. The narrow column, large font and centring forced to keep this brief. Which I think is perfectly fine for this stage of development.

**Comment by Beanow** [comment]
> I can give you a PNGCRUSHED version of the meme image, if you want.

Good idea, and something I should learn to do.
I've tried running these two images with `pngcrush -reduce [old] [new]` and got 9.7% and 7.8% reductions out of them. Does that seem right?

**Edit:** looking at the commic strip one, it shows jpeg artifacts while being a png. When the original jpg on their site doesn't at half the size. Going to replace that.

**Comment by nothingismagick** [comment]
Try using the --brute flag.

>

**Review by nothingismagick — APPROVED** [review]

**Comment by Beanow** [comment]
An additional `0.08% filesize reduction` with -brute. Why not?

**Pull request merged**